### PR TITLE
Ft make translation easy 157110620

### PR DESCRIPTION
**Pull request by younggeeks**
###  What does this PR do?
Make translations of exercises and ingredients easier 

###  Description of Task to be completed?
Users can now select which language to they wish to use.
### How should this be manually tested?
After setting up the project , User can simply click on the flag which will open dropdown of all the languages supported
### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[#157110620 Make translations of exercises and ingredients easier](https://www.pivotaltracker.com/story/show/157110620) 

Screenshots (if appropriate)
screen shot 2018-05-11 at 11 55 59

screen shot 2018-05-10 at 16 52 41

### CODE REVIEW

**Comment by mirr254** [comment]
Kindly ensure all your checks are passing

**Comment by mirr254** [comment]
Why do you have 60 files changed? Kindly explain

**Comment by mirr254** [comment]
Your screenshot doesn't reflect the drop down for changing language when creating exercises

File: wger/nutrition/views/ingredient.py [hunk 1]
```diff
@@ -175,20 +176,20 @@ class IngredientCreateView(IngredientMixin, CreateView):
     sidebar = 'ingredient/form.html'
 
     def form_valid(self, form):
-
+        print("we are down hre trying to check whether form is valid")
```
Comment (MarloneA) [inline wger/nutrition/views/ingredient.py]: Remove debug statements

**Pull request merged**